modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,262 @@
+// issue # is unknown.
+
+* On IBM Power Systems, IBM Z, and LinuxONE, the `s2i-dotnet` cluster task is unsupported.
+// issue # is unknown.
+
+* Before you install tasks based on Tekton Catalog on IBM Power Systems (ppc64le), IBM Z, and LinuxONE(s390x) using `tkn hub`, verify if the task can be executed on these platforms. To check if `ppc64le` and `s390x` are listed in the "Platforms" section of the task information, you can run the following command: `tkn hub info task <name>`
+// issue # is unknown.
+
+* You cannot use the `nodejs:14-ubi8-minimal` image stream because doing so generates the following errors:
++
+[source,terminal]
+----
+STEP 7: RUN /usr/libexec/s2i/assemble
+/bin/sh: /usr/libexec/s2i/assemble: No such file or directory
+subprocess exited with status 127
+subprocess exited with status 127
+error building at STEP "RUN /usr/libexec/s2i/assemble": exit status 127
+time="2021-11-04T13:05:26Z" level=error msg="exit status 127"
+----
+// https://issues.redhat.com/browse/SRVKP-1782
+
+
+[id="fixed-issues-1-6_{context}"]
+== Fixed issues
+
+* The `tkn hub` command is now supported on IBM Power Systems, IBM Z, and LinuxONE.
+// issue # is unknown.
+
+//[id="fixed-cli-0-21-0-1-6_{context}"]
+//=== CLI
+
+* Before this update, the terminal was not available after the user ran a `tkn` command, and the pipeline run was done, even if `retries` were specified. Specifying a timeout in the task run or pipeline run had no effect. This update fixes the issue so that the terminal is available after running the command.
+//  (link:https://github.com/tektoncd/cli/issues/1459[#1459])
+
+* Before this update, running `tkn pipelinerun delete --all` would delete all resources. This update prevents the resources in `RUNNING` state from getting deleted.
+//  https://issues.redhat.com/browse/SRVKP-1638
+
+* Before this update, using the `tkn version --component=<component>` command did not return the component version. This update fixes the issue so that this command returns the component version.
+//  (https://github.com/tektoncd/cli/pull/1408[#1408])
+
+* Before this update, when you used the `tkn pr logs` command, it displayed the pipelines output logs in the wrong task order. This update resolves the issue so that logs of completed `PipelineRuns` are listed in the appropriate `TaskRun` execution order.
+//  (link:https://github.com/tektoncd/cli/pull/1385[#1385])
+
+//[id="fixed-pipelines-operator-1-6_{context}"]
+//=== {pipelines-title} Operator
+
+* Before this update, editing the specification of a running pipeline might prevent the pipeline run from stopping when it was complete. This update fixes the issue by fetching the definition only once and then using the specification stored in the status as the source of truth. This change reduces the probability of a race condition when a `PipelineRun` or a `TaskRun` refers to a `Pipeline` or `Task` that changes while it is running.
```

Comment:
in the status as the source of truth.
Suggest ending the sentence at 'in the status'
source of truth is jargon and could be difficult to translate.